package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/button/react-simple-scroll#readme",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
+    "react": "^16.11.0",
```

Comment:
Ideally we should support the broadest range of react versions we can. Why did you choose `^16.11.0`?

Reply:
Ah yes, you're right- I'll just keep the current start range but bump the end range from `15` to `16`

Side note though, because we're now using `enzyme 3+` in our tests, they require adapters depending on the version of React you're using (in this case it'll be `16`)